.app.id~

```diff
@@ -1 +0,0 @@
-79e0800fba37e7abc532724f409b72be
```

Comment:
Use the `--amend` flag to actually merge with the previous commit. We'll rebase/squash this pull request so it doesn't show up.